Standalone NSFS all S3 ops fails on NamespaceFS.set_cur_fs_account_config

### Environment info
- NooBaa Version: **master-20210427**
- Platform: **Standalone**

### Actual behavior
1. List objects fails
```
Apr-27 11:27:11.496 [Endpoint/86794] [ERROR] core.endpoint.s3.s3_rest:: S3 ERROR <?xml version="1.0" encoding="UTF-8"?><Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><Resource>/favicon.ico</Resource><RequestId>knzrnd3c-16ao4g-1bmw</RequestId></Error> GET /favicon.ico {"host":"localhost:6001","connection":"keep-alive","sec-ch-ua":"\" Not A;Brand\";v=\"99\", \"Chromium\";v=\"90\", \"Google Chrome\";v=\"90\"","sec-ch-ua-mobile":"?0","user-agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.85 Safari/537.36","accept":"image/avif,image/webp,image/apng,image/svg+xml,image/*,*/*;q=0.8","sec-fetch-site":"same-origin","sec-fetch-mode":"no-cors","sec-fetch-dest":"image","referer":"http://localhost:6001/images/","accept-encoding":"gzip, deflate, br","accept-language":"en-US,en;q=0.9,he;q=0.8"} Error: Invalid Request
    at NamespaceFS.set_cur_fs_account_config (/snapshot/noobaa-core-20210427-80602-1didalo/src/sdk/namespace_fs.js:128:37)
    at NamespaceFS.list_objects (/snapshot/noobaa-core-20210427-80602-1didalo/src/sdk/namespace_fs.js:164:40)
    at ObjectSDK.list_objects (/snapshot/noobaa-core-20210427-80602-1didalo/src/sdk/object_sdk.js:339:19)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Object.get_bucket [as handler] (/snapshot/noobaa-core-20210427-80602-1didalo/src/endpoint/s3/ops/s3_get_bucket.js:36:19)
    at async handle_request (/snapshot/noobaa-core-20210427-80602-1didalo/src/endpoint/s3/s3_rest.js:183:19)
    at async Object.s3_rest [as handler] (/snapshot/noobaa-core-20210427-80602-1didalo/src/endpoint/s3/s3_rest.js:102:9)
```

### Expected behavior
1. Should work.

### Steps to reproduce
1. List objects.

### More information - Screenshots / Logs / Other output

--- a/src/core/nsfs.js
+++ b/src/core/nsfs.js
@@ -37,8 +37,10 @@ Arguments:
 const OPTIONS = `
 Options:
 
-    --http_port <port>     (default 6001)   Set the S3 endpoint listening HTTP port to serve.
-    --https_port <port>    (default 6443)   Set the S3 endpoint listening HTTPS port to serve.
+    --http_port <port>     (default 6001)           Set the S3 endpoint listening HTTP port to serve.
+    --https_port <port>    (default 6443)           Set the S3 endpoint listening HTTPS port to serve.
+    --uid <uid>            (default process uid)    Send requests to the Filesystem with uid.
+    --gid <gid>            (default process gid)    Send requests to the Filesystem with gid.
 `;
 
 const WARNINGS = `
@@ -67,6 +69,12 @@ async function main(argv = minimist(process.argv.slice(2))) {
         const https_port = Number(argv.https_port) || 6443;
         const fs_root = argv._[0];
         if (!fs_root) return print_usage();
+
+        let fs_config = {
+            uid: Number(argv.uid) || process.getuid(),
+            gid: Number(argv.gid) || process.getgid(),
+            backend: ''
+        };
 
         if (!fs.existsSync(fs_root)) {
             console.error('Error: Root path not found', fs_root);
@@ -102,6 +110,9 @@ async function main(argv = minimist(process.argv.slice(2))) {
         object_sdk.read_bucket_sdk_caching_info = noop;
         object_sdk.read_bucket_sdk_policy_info = noop;
         object_sdk.read_bucket_usage_info = noop;
+        object_sdk.requesting_account = {
+            nsfs_account_config: fs_config
+        };
 
         const endpoint = require('../endpoint/endpoint');
         await endpoint.start_endpoint({

--- a/src/util/nb_native.js
+++ b/src/util/nb_native.js
@@ -63,7 +63,8 @@ async function read_rand_seed(seed_bytes) {
     if (process.platform === 'win32') return;
     let fh;
     const clean_fh = async () => {
-        if (!fh) return;
+        // Ignore seed in standalone due to pkg issue: https://github.com/noobaa/noobaa-core/issues/6476
+        if (!fh || Number.isInteger(fh)) return;
         console.log('read_rand_seed: closing fd ...');
         try {
             await fh.close();
@@ -81,6 +82,8 @@ async function read_rand_seed(seed_bytes) {
             if (!fh) {
                 console.log(`read_rand_seed: opening ${random_dev} ...`);
                 fh = await fs.promises.open(random_dev, 'r');
+                // Ignore seed in standalone due to pkg issue: https://github.com/noobaa/noobaa-core/issues/6476
+                if (Number.isInteger(fh)) break;
             }
             console.log(`read_rand_seed: reading ${count} bytes from ${random_dev} ...`);
             const { bytesRead } = await fh.read(buf, offset, count, null);

